Training3/src/tasks/src/App.js

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import Modal from "./Modal";
+
+function App() {
+  const [isOpen, setIsOpen] = useState(false);
+  function handleClick() {
+    setIsOpen(!isOpen);
+  }
+  useEffect(() => {
+    if (isOpen) {
+      window.addEventListener("keypress", (e) => {
+        var name = e.key;
+        alert(`Key pressed: ${name}`);
+      });
+    } else {
+      window.removeEventListener("keypress", () => {});
```

Comment:
you must using return function in useEffect to take the variable above

Reply:
@phuongluongZigvy  You put the  window.removeEventListener, it should ínide if (isOpen), we only want clean remove listener when the effect (isOpen) is cleaned

Reply:
Fix it by the same way as above comment